src/core/border_router/routing_manager.hpp

```diff
@@ -478,6 +478,7 @@ class RoutingManager : public InstanceLocator
     void HandleDiscoveredPrefixTableChanged(void);
     bool NetworkDataContainsOmrPrefix(const Ip6::Prefix &aPrefix) const;
     void UpdateRouterAdvertHeader(const Ip6::Nd::RouterAdvertMessage *aRouterAdvertMessage);
+    bool IsReceivdRouterAdvertFromManager(const Ip6::Nd::RouterAdvertMessage &aRaMessage) const;
```

Comment:
`IsReceivdRouterAdvertFromManager-> IsReceivedRouterAdvertFromManager`
 is it a typo or intentional?

Reply:
It is a type